File: user_spring/src/main/java/com/kjj/user/dto/user/PolicyDto.java
package com.kjj.user.dto.user;

import com.kjj.user.entity.UserPolicy;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class PolicyDto {
    private boolean monday;
    private boolean tuesday;
    private boolean wednesday;
    private boolean thursday;
    private boolean friday;

    private Long defaultMenu;

    public static PolicyDto from(UserPolicy policy) {
        return new PolicyDto(
                policy.isMonday(),
                policy.isTuesday(),
                policy.isWednesday(),
                policy.isThursday(),
                policy.isFriday(),
                policy.getDefaultMenu()
        );
    }
}

File: user/src/main/kotlin/com/kjj/user/dto/user/PolicyDto.kt
package com.kjj.user.dto.user

import com.kjj.user.entity.UserPolicy

data class PolicyDto(
    val monday: Boolean,
    val tuesday: Boolean,
    val wednesday: Boolean,
    val thursday: Boolean,
    val friday: Boolean,
    val defaultMenu: Long?
) {
    companion object {
        fun from(policy: UserPolicy): PolicyDto {
            return PolicyDto(
                policy.monday,
                policy.tuesday,
                policy.wednesday,
                policy.thursday,
                policy.friday,
                policy.defaultMenu
            )
        }
    }
}